src/inputs/Switch/Switch.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import './Switch.module.css'
```

Comment:
**issue:** should be `import * as styles from './Switch.module.css'`, because in that case you don't use power of CSS modules. I mean - your class looks like this (screen 1). If you have other css with the same class you will overwrite your component styles. That's why we do `className={styles.switch}`, so unique classname is being generated, and you will avoid such scenarios.

<img width="464" alt="Zrzut ekranu 2023-02-4 o 16 33 12" src="https://user-images.githubusercontent.com/22893188/216775940-a0d7bbbe-45af-4cbb-b8d6-5a45ad1a382a.png">
<img width="917" alt="Zrzut ekranu 2023-02-4 o 16 34 30" src="https://user-images.githubusercontent.com/22893188/216775998-4fa66b05-f4fe-481c-96a4-407349bd033b.png">